.github/workflows/test.yml

```diff
@@ -10,39 +10,39 @@ on:
       # - `pull_request` see below
       # - https://github.com/prisma/prisma/blob/master/src/.buildkite/test/buildkite-entry.sh
       # - https://github.com/prisma/prisma/blob/master/src/.buildkite/publish/buildkite-entry.sh
-      - '*.md'
-      - '.vscode/**'
-      - '.github/ISSUE_TEMPLATE/**'
-      - 'docs/**'
-      - 'examples/**'
-      - 'LICENSE'
-      - '.github/CODEOWNERS'
-      - 'src/.buildkite/**'
-      - 'src/.dockerignore'
-      - '*.bench.ts'
-      - 'src/scripts/ci/publish.ts'
-      - 'src/graphs/**'
+      - "*.md"
+      - ".vscode/**"
+      - ".github/ISSUE_TEMPLATE/**"
+      - "docs/**"
+      - "examples/**"
+      - "LICENSE"
+      - ".github/CODEOWNERS"
+      - "src/.buildkite/**"
+      - "src/.dockerignore"
+      - "*.bench.ts"
+      - "src/scripts/ci/publish.ts"
+      - "src/graphs/**"
   pull_request:
     paths-ignore:
       # Any update here needs to be done for
       # - `push`see before
       # - https://github.com/prisma/prisma/blob/master/src/.buildkite/test/buildkite-entry.sh
       # - https://github.com/prisma/prisma/blob/master/src/.buildkite/publish/buildkite-entry.sh
-      - '*.md'
-      - '.vscode/**'
-      - '.github/ISSUE_TEMPLATE/**'
-      - 'docs/**'
-      - 'examples/**'
-      - 'LICENSE'
-      - '.github/CODEOWNERS'
-      - 'src/.buildkite/**'
-      - 'src/.dockerignore'
-      - '*.bench.ts'
-      - 'src/scripts/ci/publish.ts'
-      - 'src/graphs/**'
+      - "*.md"
+      - ".vscode/**"
+      - ".github/ISSUE_TEMPLATE/**"
+      - "docs/**"
+      - "examples/**"
+      - "LICENSE"
+      - ".github/CODEOWNERS"
+      - "src/.buildkite/**"
+      - "src/.dockerignore"
+      - "*.bench.ts"
+      - "src/scripts/ci/publish.ts"
+      - "src/graphs/**"
 
 env:
-  PRISMA_TELEMETRY_INFORMATION: 'prisma test.yml'
+  PRISMA_TELEMETRY_INFORMATION: "prisma test.yml"
```

Comment:
Are we sure yml is fine with these changes?

Reply:
Let me revert the changes, prettier......